src/main/kotlin/frc/robot/Robot.kt

```diff
@@ -95,9 +95,7 @@ object Robot : LoggedRobot() {
         autonomousCommand = robotContainer!!.getAutonomousCommand()
 
         // Schedule the autonomous command
-        if (autonomousCommand != null) {
-            autonomousCommand!!.schedule()
-        }
+        autonomousCommand!!.schedule()
```

Comment:
That's cool

Reply:
This is a redundant check. The getAutonomousCommand() method returns a 'Command' type, not a nullable 'Command?' type.

Reply:
The code is just very excited about autonomous